src/ProductsXmlFeed.php

```diff
@@ -168,7 +168,7 @@ private static function get_attributes_xml( $product, $indent ) {
 		$xml               = '';
 
 		// Merge with parent's attributes if it's a variation product.
-		if ( $product instanceof WC_Product_Variation ) {
+		if ( $product instanceof WC_Product_Variation && ! empty( $product->get_parent_id() ) ) {
```

Comment:
Instead of checking `! empty( $product->get_parent_id() )`, I think it would be better to check the value of `$parent_product` from the next line. A parent ID can be non-empty (therefore passing the if statement), yet still point to an invalid product (therefore triggering `wc_get_product()` to return a non-object).